apptools/preferences/ui/widget_editor.py

```diff
@@ -58,14 +58,14 @@ def update_editor(self):
     # Private interface.
     ###########################################################################
 
-    def _on_page_changed(self, obj, trait_name, old, new):
+    def _on_page_changed(self, event):
```

Comment:
Maybe rename to `_handle_change_to_selected_page`?

Also, i'm not sure why we're not using the decorator here,

Reply:
It's quite possible that this code was written before the decorator existed in traits - i don't see a good reason why we can't use the new decorator here.